zh/initialize-cluster.md

```diff
@@ -11,23 +11,13 @@ category: how-to
 >
 > 以下功能只在第一次创建集群时有作用，集群创建之后再设置或修改不会生效。
 
-## 设置初始化账号和密码
+请参考 TidbInitializer [示例](https://github.com/pingcap/tidb-operator/blob/master/manifests/initializer/tidb-initializer.yaml)和 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html)（示例和 API 文档请切换到当前使用的 TiDB Operator 版本）以及下面的步骤，完成 TidbInitializer CR，并保存到文件 `<cluster-name>/tidb-initializer.yaml`。
 
-集群创建时默认会创建 `root` 账号，但是密码为空，这会带来一些安全性问题。可以通过如下步骤为 `root` 账号设置初始密码：
+1. 初始化账号和密码设置
```

Comment:
How about using a second-level heading like this? It helps make the content look clearer.
```suggestion
## 初始化账号和密码设置
```